test/e2e/kubectl.go

```diff
@@ -818,6 +819,7 @@ var _ = Describe("Kubectl client", func() {
 		})
 
 		AfterEach(func() {
+			glog.Info(runKubectl("get", "events", "-a"))
```

Comment:
maybe check if reportDir is nonempty, and if so, save the events in a log file in there?

that way we could merge this and let the regular CI flake out the test, rather than trying to get k8s-bot to do it.